components/schemas/stacks/spec/StackContainerConfigIntegrations.yml

```diff
@@ -80,27 +90,32 @@ properties:
       - restore
       - retention
     properties:
-      destination:
-        type: string
+      integration_id:
+        $ref: ../../HybridIdentifier.yml
+        description: An identifier of an integration that supports backups. All backups will be sent to this destination.
```

Comment:
Im not sure how this works exactly I think im out of the loop, is there an example we can add here?